src/PhpSpreadsheet/Reader/IReadFilter.php

```diff
@@ -9,9 +9,8 @@ interface IReadFilter
      *
      * @param string $column Column address (as a string value like "A", or "IV")
      * @param int $row Row number
-     * @param string $worksheetName Optional worksheet name
      *
      * @return bool
      */
-    public function readCell($column, $row, $worksheetName = '');
+    public function readCell($column, $row);
```

Comment:
This is a major breaking change in public interfaces and should be reverted, along with all other implementing classes